test/support/index.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
+const path = require('path');
 
 process.env.NODE_ENV = 'test';
-
+process.env.SETTINGS_PATH = process.env.SETTINGS_PATH || path.join(__dirname, '../..', 'config', 'settings.yml');
```

Comment:
good point @kamath-prasad  :)

Reply:
yes. it may. better to remove it?

Reply:
No. This will be used only during test runs.  During tests, we never use my-settings.yml as it could have things which would break tests. So I dont see any harm in this.

Reply:
Ok. Understood.